src/components/post-card.astro

```diff
@@ -36,14 +36,21 @@ const author = await getEntry(
     </div>
     <div class='w-full p-4 flex flex-col gap-3 @3xl:gap-6 @3xl:pr-20'>
       <div class='flex items-center gap-1.5'>
-        <img
-          class='rounded-full'
-          width='18'
-          height='18'
-          alt={`A photo of ${author.data.name}`}
-          src={author.data.avatar.src}
-        />
-        <p class='text-sm @3xl:text-lg text-tertiary'>{author.data.username}</p>
+        <a
+          href={`/authors/${post.data.author.slug}`}
+          class='flex items-center gap-1.5 z-10 hover:border-2 border-link p-2 rounded-md'
```

Comment:
Let's remove the `hover:border-2 border-link p2 rounded-md` classes. Just change the text color to `link` and show an underline on hover.